src/components/users/views.tsx

```diff
@@ -101,3 +121,168 @@ export function UserPage(props: IUserPageProps): ReactElement {
     </>
   );
 }
+
+export function PasswordResetRequest(props: IPasswordResetFormProperties): ReactElement {
+  return <div className="govuk-grid-row">
+    <div className="govuk-grid-column-two-thirds">
+      <h1 className="govuk-heading-l">Password reset</h1>
+      <p className="govuk-body">
+        We will send you an activation link via email. If you do not receive an
+        email in the next 24 hours, your account may not exist in the system or
+        you could have setup authentication with Google or Microsoft Single
+        Sign-on.
+      </p>
+
+      {props.invalidEmail ? <div
+        className="govuk-error-summary"
+        aria-labelledby="error-summary-title"
+        role="alert"
+        tabIndex={-1}
+        data-module="govuk-error-summary"
+      >
+        <h2 className="govuk-error-summary__title" id="error-summary-title">
+          Error validating the request
+        </h2>
+
+        <div className="govuk-error-summary__body">
+          <ul className="govuk-list govuk-error-summary__list">
+            <li>
+              <a href="#email">Valid email address must be provided</a>
+            </li>
+          </ul>
+        </div>
+      </div> : <></>}
+
+      <form method="post" className="govuk-!-mt-r6">
+        <input type="hidden" name="_csrf" value={props.csrf} />
+
+        <div className={`govuk-form-group ${props.invalidEmail ? 'govuk-form-group--error' : ''}`}>
+          <label className="govuk-label" htmlFor="email">
+            Email address
+          </label>
+
+          {props.invalidEmail ? <span id="email-error" className="govuk-error-message">
+            <span className="govuk-visually-hidden">Error:</span>{' '}
+            Valid email address must be provided
```

Comment:
can we update the error message, something like https://design-system.service.gov.uk/patterns/email-addresses/#error-messages